src/bun.js/api/server.zig

```diff
@@ -2945,14 +2960,18 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
             this: *RequestContext,
             value: JSC.JSValue,
             status: u16,
+            request_value: JSValue,
         ) void {
             JSC.markBinding(@src());
             if (!this.server.config.onError.isEmpty() and !this.flags.has_called_error_handler) {
                 this.flags.has_called_error_handler = true;
                 const result = this.server.config.onError.callWithThis(
                     this.server.globalThis,
                     this.server.thisObject,
-                    &.{value},
+                    &.{
+                        value,
+                        request_value,
```

Comment:
Can you add an assertion that `request_value != .zero`? If we pass .zero it will crash with a much weirder error